samri/pipelines/fc.py

```diff
@@ -182,6 +182,7 @@ def seed_based_connectivity(ts, seed_mask,
 	return seed_based_correlation_img
 
 def correlation_matrix(ts,
+        confounds,
```

Comment:
Is it mandatory to have confounds? If not please set it to be None by default, and make the following confound processing conditional. If yes, please put it on the first row, along with the other positional arguments.